site/en/docs/privacy-sandbox/attribution-reporting-data-clearing/index.md

```diff
@@ -48,8 +48,8 @@ However, they likely won't be consistently observable for all organizations
 that use the API.
 
 These numbers are aggregated and measured for the set of sites that were running an origin
-trial. They differ across sites and are likely to change as more organizations use the API in an
-origin trial and use different parameters.
+trial. They differ across sites and are likely to change as more organizations have used the API in an
+origin trial since this writing and use different parameters.
```

Comment:
might be worth saying when the time is specifically instead of this way of phrasing? not too strong of an opinion though

Reply:
ya you're right. I need to re re-word:)